utils/build-script-impl

```diff
@@ -810,6 +813,14 @@ if [[ "${CROSS_COMPILE_HOSTS}" ]]; then
     SKIP_COMPILER_RT=1
 fi
 
+if [[ "${BUILD_SWIFT_STDLIB}" ]]; then
+    BUILD_SWIFT_DYNAMIC_STDLIB=1
```

Comment:
Shouldn't it turn off the static one as well?

Also, would you mind moving the alias processing into Python?  (Possibly in a future change.)